File: Benchmarks/normal/python/django/testcode/benchmark_test_01038.py
# SPDX-License-Identifier: Apache-2.0
from django.http import JsonResponse


def BenchmarkTest01038(request, path_param):
    path_value = path_param
    data = '%s' % (path_value,)
    return JsonResponse({'error': 'An internal error occurred'}, status=500)

File: Benchmarks/normal/java/spring/testcode/BenchmarkTest01038.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest01038 {

    private static String normalize(String v) { return v.strip(); }

    @GetMapping("/BenchmarkTest01038")
    public void BenchmarkTest01038(@RequestHeader("User-Agent") String userAgent, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String uaValue = userAgent != null ? userAgent : "";
        String data = normalize(uaValue);
        response.sendError(500, data);
    }
}
